examples/wallet/app/actions/router.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import { push } from 'connected-react-router';
+import type { Dispatch, GetState } from '../reducers/types';
+import type { Address } from '../models';
+import routes from '../constants/routes';
+
+// eslint-disable-next-line
```

Comment:
A disable for the specific rule should perhaps be used.
The broken rule is [prefer-default-export](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md), since the module has only one export and is not a default.
This is because I'm lazy and would rather not modify imports of this function when adding a second one 🙃.